app/config/init.js

```diff
@@ -36,6 +36,8 @@ const _init = function (cfg) {
       notify('Error reading configuration: `config` key is missing');
       return _extractDefault(cfg.defaultCfg);
     }
+    // Ignore undefined values in plugin array Issue #1862
+    _cfg.plugins = _cfg.plugins.filter(Boolean);
```

Comment:
Could `_cfg.plugins` be `undefined`? If yes we should check it:
```javascript
_cfg.plugins = (_cfg.plugins && _cfg.plugins.filter(Boolean)) || [];
```
`localPlugins` should be filtered too